libraries/Adapters/Microsoft.Bot.Builder.Adapters.Slack/SlackAdapter.cs

```diff
@@ -362,7 +367,15 @@ public async Task ProcessAsync(HttpRequest request, HttpResponse response, IBot
                                 {
                                     Id = null,
                                 },
-                                ChannelData = slackEvent["event"],
+                                ChannelData = new NewSlackMessage()
+                                {
+                                    type = slackEvent["event"].type,
```

Comment:
Instead of calling `slackEvent["event"]` several times, wouldn't it be better to have a variable with its value?

Reply:
plusone